src/functionaltests/resources/features/subscriptionNotification.feature

```diff
@@ -22,3 +22,14 @@ Feature: Test Subscription Trigger
     When I send one previous event again
     And Wait for EI to aggregate objects
     And Failed notification db should contain 2 objects
+
+  @AddedDefaultValuesSenderAndSubject

```

Comment:
This test could be extended to test also when one of the options is given but not the other, thus the given option should not be replaced with default.

Reply:
I agree, a new sceneario in which we test that the configured properties are used and not default values ... I don't see any mention of which configuration is used in any of the features. The first scenario "SubscriptionNotificationOK" could be extended with an assert that the email notifications has the correct subject + sender (matching input configuration)

Reply:
In the first scenario I have extended assert for subject even. so should I remove the new scenario?

Reply:
Ah my mistake, yes I see that you extended the first scenario - good! 

Just an improvement tip: the feature for the new scenario "Test default values mail notification" there is no mention of how the configuration affects these values (i.e. first scenario we provide email configuration so EI will use those for notifications, and in next scenario we provide no configuration so EI will use default values instead) 

I imagine something like this (with the first scenario being the existing one just extended:

**Scenario 1**
Given the REST API is up and running
And email configuration is provided
We create subscriptions 
etc ...
Notification email contains \<our expected\> values

**Scenario 2**
Given the REST API is up and running
And email configuration is NOT provided
We create subscriptions 
etc ...
Notification email contains default values